test/source/tests/gmail.ts

```diff
@@ -352,6 +353,16 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       await gmailPage.click('[aria-label^="Discard draft"]');
     }));
 
+    ava.default('mail.google.com - Outlook encrypted message with attachment is recognized', testWithBrowser(undefined, async (t, browser) => {
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'ci.tests.gmail@flowcrypt.dev');
+      await SetupPageRecipe.manualEnter(settingsPage, 'flowcrypt.compatibility.1pp1', { submitPubkey: false, usedPgpBefore: true, },
+        { isSavePassphraseChecked: false, isSavePassphraseHidden: false });
+      const gmailPage = await openGmailPage(t, browser);
+      await gotoGmailPage(gmailPage, '/FMfcgzGllVqqBbjHQQRDsSwcZBlMRzDr');
+      expect(await gmailPage.isElementPresent('@attachments')).to.equal(false);
```

Comment:
We should make sure that we're waiting for the loading to complete.
Need to double-check that this test actually fails if we don't mark ATT00001 as 'hidden'

Reply:
Yes, sir.

> Need to double-check that this test actually fails if we don't mark ATT00001 as 'hidden'

I can also confirm that the test above in `gmail.ts` is failing when the changes from `gmail-parser.ts` are reverted to the original - Please correct me if I'm wrong, Is that the right approach to double-check?

Reply:
Yes, that's the right approach.
I guess for gmail we have to use `sleep`

Reply:
Thank you for the confirmation, sir. I have retained using the`sleep` function on the gmail test.